site/src/components/LinkboxList.js

```diff
@@ -21,7 +21,7 @@ const LinkboxList = ({ data, className }) => (
             external={item.external}
             heading={item.name}
             text={item.text}
-            imgProps={item.imgProps ? { ...item.imgProps, src: withPrefix(item.imgProps.src) } : undefined}
+            imgProps={{ src: withPrefix("/images/components/overview/" + item.name + "@2x.png"), width: 280, height: 180}}
```

Comment:
Maybe we could add the imageUrl in the ComponentsList component configuration? 
I think we need an alternative text for the image too

Reply:
The format could be: 
```js
 const componentData = [
    {
      name: 'Accordion',
      text: 'Accordions can be used to hide and reveal information.',
      linkboxAriaLabel: 'Accordion component',
      linkAriaLabel: 'Go to the Accordion component page',
      href: '/components/accordion',
      imageProps: { 
         src: '/images/components/overview/Accordion@2x.png', 
         altText: "An illustration of Accordion component"
      }
    },
```

Reply:
Thanks for pointing that out! Now I understand the structure and the dependencies of the site better.

Reply:
The change might be tricky but we probably need to map the component data in ComponentsList like this:
```js
const componentDataWithImageProps = componentData.map(component => 
  ({...component, 
     imgProps: { 
       src: '/images/components/overview/" + item.name + "@2x.png', 
       alt: `An illustration of ${component.name} component`
     }
  })
)
return <LinkboxList data={componentDataWithImageProps} />;

```
And the make the LinkBoxList to accept imgProp.alt if there is one.

Reply:
Thanks! 👍 I added imgProps to the component data in ComponentsList and restored LinkboxList back to the way it was. Now the image URLs on the front page work fine and the component overview page shows the images and their alternative texts correctly.

Reply:
Still wondering one more thing. Should I add the height and width to imgProps in ComponentList like in FrontPageLinkList (see below)?

`{
      name: 'Components',
      text: 'UI components ready to use for complete layouts',
      linkboxAriaLabel: 'Components',
      linkAriaLabel: 'Go to the Components page',
      href: '/components',
      imgProps: {
        src: '/images/components/components.svg',
        height: 198,
        width: 300,
      },
    },`

Reply:
Yes, it would make sense to add those too 👍